packages/cli/src/commands/capabilities/presentation.ts

```diff
@@ -82,19 +82,17 @@ export default class PresentationsCommand extends APIOrganizationCommand<typeof
 	static args = capabilityIdOrIndexInputArgs
 
 	async run(): Promise<void> {
-		const idOrIndex = this.args.version
-			? { id: this.args.id, version: this.args.version }
-			: this.args.id
-		const sortKeyName = 'id'
-		const config: OutputItemOrListConfig<CapabilityPresentation, CapabilitySummaryWithNamespace> = {
-			primaryKeyName: 'id',
-			sortKeyName,
-			listTableFieldDefinitions: ['id', 'version', 'status'],
-			buildTableOutput: (data: CapabilityPresentation) => buildTableOutput(this.tableGenerator, data),
+
+		const capabilityId = await chooseCapability(this, this.args.id, this.args.version, undefined, this.flags.namespace)
+		try {
+			const presentation = await this.client.capabilities.getPresentation(capabilityId.id, capabilityId.version)
+			await formatAndWriteItem(this, { buildTableOutput: data => buildTableOutput(this.tableGenerator, data) }, presentation)
+		} catch (error) {
+			if ('message' in error && error.message.includes('status code 403')) {
```

Comment:
I usually avoid special error handling of errors from the API. The API returns error messages in a standard format (most of the time). We have an issue open to "display standard error messages in a nicer format" which would do something akin to this, but generically for all commands and errors.

Also, if you do want to check for a status code, you should be able to find it more specifically at `error.response.status` like this:

```if (error.response?.status === 403)```

Reply:
I've taken out the special handling.  I really prefer having a nicer format to the message, but I see your point that it's better handled uniformly via that other open issue.